tests/controllers/hibp.test.js

```diff
@@ -9,6 +9,32 @@ const { testBreaches } = require("../test-breaches");
 require("../resetDB");
 
 
+test("notify POST without token should throw error", async() => {
+  const testEmail = "victim@spoofattack.com";
+  const testHash = sha1(testEmail);
+  const testPrefix = testHash.slice(0, 6).toUpperCase();
+  const testSuffix = testHash.slice(6).toUpperCase();
+
+  const mockRequest = { body: { hashPrefix: testPrefix, hashSuffixes: [testSuffix], breachName: "SomeSensitiveBreach" } };
+  const mockResponse = { status: jest.fn(), json: jest.fn() };
+
+  await expect(hibp.notify(mockRequest, mockResponse)).rejects.toThrow("HIBP notify endpoint requires valid authorization token.");
+});
+
+
+test("notify POST with invalid token should throw error", async() => {
+  const testEmail = "victim@spoofattack.com";
+  const testHash = sha1(testEmail);
+  const testPrefix = testHash.slice(0, 6).toUpperCase();
+  const testSuffix = testHash.slice(6).toUpperCase();
```

Comment:
I keep thinking we should create some wrapper function that takes an email address and returns an object w/ the `testHash`, `testPrefix`, and `testSuffix` since I think we're using it 3-4x in this file. But it's more of a small nit since I think this is the only file that separates out the prefix/suffix.

```js
function doStuff(testEmail) {
  const testHash = sha1(testEmail);
  return {
	testHash,
    testPrefix: testHash.slice(0, 6).toUpperCase(),
    testSuffix: testHash.slice(6).toUpperCase();
  };
}

// ...

const testEmail = "victim@spoofattack.com";
const {testHash, testPrefix, testSuffix} = doStuff(testEmail);
```